src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/resource/UserManagementResource.kt

```diff
@@ -35,6 +35,49 @@ class UserManagementResource(
   private val userService: UserService,
 ) {
 
+  @PreAuthorize("hasRole('ROLE_MANAGE_NOMIS_USER_ACCOUNT')")
+  @PutMapping("/{username}/logon-date")
+  @ResponseStatus(HttpStatus.OK)
+  @Operation(
+    summary = "Record the date and time of last logon for the user",
+    description = "Record the date and time of last logon for the user. Requires role ROLE_MANAGE_NOMIS_USER_ACCOUNT",
+    security = [SecurityRequirement(name = "MANAGE_NOMIS_USER_ACCOUNT")],
```

Comment:
we had a PR against our repo when we referenced security requirements - think they also need referencing in the OpenApi configuration.  See https://github.com/ministryofjustice/hmpps-prisoner-search/pull/155

Reply:
I'll look at this as a follow up and correct it across the board: https://dsdmoj.atlassian.net/browse/HAAR-1967